app/serializers/api/admin/customer_serializer.rb

```diff
@@ -1,5 +1,6 @@
 class Api::Admin::CustomerSerializer < ActiveModel::Serializer
   attributes :id, :email, :enterprise_id, :user_id, :code, :tags, :tag_list, :name
+  attributes :allow_charges, :default_card_present?
```

Comment:
`:allow_charges` should go in #2275 , right?

Reply:
Oh, yes, good catch, I wonder how that got there...